libs/utils/env.py

```diff
@@ -644,6 +644,17 @@ def _init_platform(self):
         # Adding topology information
         self.platform['topology'] = self.topology.get_level("cluster")
 
+        # Adding kernel build information
+        kver = self.target.execute('uname -r').strip()
+        match = KVERSION_RE.match(kver)
+        if match:
+            self.platform['kernel_version'] = match.group('ver')
```

Comment:
Might be handy to have this as a tuple of ints (like `(3, 18)` instead of `"3.18"`), then we would leave space for richer info (e.g. stable kernels like 4.9.10) in the future. This works nicely because tuples sort lexicographically, so in the later commit you could have `if self.platform.get('kernel_version', (0,)) > (4, 7):`